test/util/base/UnitSpec.scala

```diff
@@ -18,5 +18,9 @@ package base
 
 import org.scalatest.{MustMatchers, WordSpec}
 import org.scalatestplus.mockito.MockitoSugar
+import reactivemongo.api.commands.{LastError, WriteResult}
 
-trait UnitSpec extends WordSpec with MustMatchers with MockitoSugar
+trait UnitSpec extends WordSpec with MustMatchers with MockitoSugar {
+  val dummyWriteResultSuccess: WriteResult =
```

Comment:
Why in UnitSpec ??

Reply:
We're using this in few places and every place extends `UnitSpec`. This is probably the only one existing place where we can put it. Different approach will be new trait and extending this trait in specific tests or creating this every time in specific test